config.json

```diff
@@ -805,7 +805,14 @@
       "core": false,
       "difficulty": 7,
       "slug": "clock",
-      "topics": null,
+      "topics": [
+        "classes",
+        "equality",
+        "integers",
+        "logic",
+        "object_oriented_programming",
+        "time"
```

Comment:
Great suggestions! The only ones I'm not sure object are `object_oriented_programming` and `classes`. They do apply to this exercise but they also apply to every other exercise and I'm not sure it would be useful to add them to every exercise. What do you think?

I would also add `strings` as a topic since you have to implement `toString` :)

Reply:
I've added `strings` and removed `classes` (no new class was created when solving this exercise). However, I left the `object_oriented_programming` in place, because IMHO `hashcode()` and `equals()` methods are pretty inherent to OOP in Java.

Reply:
Okay, fair enough :)